e3db/tests/test_search_integration.py

```diff
@@ -0,0 +1,142 @@
+import e3db
+import os
+import binascii
+import pytest
+import time
+import e3db.types
+import hashlib
+import requests
+from datetime import datetime
+from datetime import timedelta
+from ..auth import E3DBAuth
+from ..types import ClientDetails
+
+local_api_url = os.environ["LOCAL_API_URL"]
+local_boot_key = os.environ["LOCAL_BOOT_KEY"]
+local_boot_secret = os.environ["LOCAL_BOOT_SECRET"]
+
+class TestSearchIntegration():
+    @classmethod
+    def internal_register_client(self, public_key, private_key):
+        url = "{0}/{1}/{2}/{3}".format(local_api_url, 'v1', 'storage', 'clients')
```

Comment:
It was a method needed to test locally, because I don't know how to get a reg token from a local pds//or any pds without the dashboard (a different problem). But since we're deploying SS to dev, it will use a different method (that I will push later)

Reply:
Any updates on this using normal registration workflow?